packages/docs/docs/guide/deploy.md

```diff
@@ -13,6 +13,16 @@ The following guides are based on a few shared assumptions:
   }
 }
 ```
+## AWS Amplify
+In this guide we'll walk through how to deploy and host your Gatsby site using the [AWS Amplify Console](https://console.amplify.aws).
```

Comment:
Gatsby site？

Reply:
oops! My apologies :) I'll fix it right away.